src/core/syscalls/os_syscall_handler.rs

```diff
@@ -1097,4 +1097,33 @@ mod tests {
             })
         )
     }
+
+    #[test]
+    fn test_get_block_number() {
+        let mut syscall = OsSyscallHandler::new(
+            VecDeque::new(),
```

Comment:
A similar test to this one should exists in the file `business_logic_syscall_handler.rs` as this is one of the hints that must be executed in the hint processor, we should test that it is correctly called and handled. There is a test in the previously mention file  that tests the `emit_event` hint, a similar tests to that one should be added.